asserts/database.go

```diff
@@ -289,13 +303,26 @@ func (db *Database) Find(assertionType *AssertionType, headers map[string]string
 		}
 		keyValues[i] = keyVal
 	}
-	assert, err := db.bs.Get(assertionType, keyValues)
-	if err != nil {
-		return nil, err
+
+	var assert Assertion
+Got:
```

Comment:
`Loop` is generally a good label for those cases. `break Loop` reads very nicely.